### DISCUSSION TITLE
Some simple pages without showing menu and navigation

### DISCUSSION THREAD

**Discussion by san-sei** [thuliteio/doks#687] [Q&A] [answered]
I am looking into designing some pages that can be accessed by users when exploring the app on smartphones at some points as single-page help. For these pages (direct URL will be used) I wish to not include menu and navigation at all. So when the user is seeing it on mobile there are no other points to look at, at that time.

@h-enk I wish to create them using markdown in the same way docs or others work. Is possible to exclude the menu by any configuration on the header of markdown?

**Comment by h-enk** [comment, marked as answer]
Yes it's possible, but you'd have to customize for it yourself. It's not in the planning to make this part of Doks.

**Reply by san-sei** [reply]
I am now doing this using `<style> ... </style>` in md pages, and working as expected.
